### ISSUE TITLE
Copy of Localization: Design onboarding (+dev onboarding updates)

### ISSUE THREAD

**Issue by Charcharbinkz** [cds-snc/design-gc-conception#1725]
**Owner** : @nchoquet 

**Validation** : Design lead

**Context** : 

[Figma file](https://www.figma.com/design/nfpEteHyEJ9dhJ2YvuvLeo/Documentation-site-Component-Pages?node-id=18150-2963&p=f&t=z1O5fho5nT7Jblmy-0)

2 new pages, and a few minor updates to existing pages. 
Pages marked "existing" only need the text in pink translated. 

- [ ] content source
- [ ] audience : dev, designers, public servants, end users
- [ ] related requirement doc and decision docs
- [ ] if applicable, previous tranlsation of this if applicable
- [ ] if applicable, comparable or reference to keep in mind

### Acceptance criteria
- [ ] check if all info needed are there in context section
- [ ] research new terminology and document decision, sources, and rationale
- [ ] new term added to Phrase
- [ ] Research and provide French examples or guidance
- [ ] Produce French version
- [ ] Ensure consistent terminology
- [ ] Check for plain language and inclusive language
- [ ] adjust formatting, punctuation (if importing to Figma)
- [ ] review by design lead
- [ ] hand off for publication 

### Definition of done :
- [ ] French version of content for publishing
- [ ] Content is consistent, clear, and correct
- [ ] Decisions documented   

### Deliverable :
- [ ] Document with both content to be translated and its translation.
- [ ] If applicable, localization decisions with guidance, information architecture recommendantion and other proposed actions.

**Comment by mcman12** [comment]
@Charcharbinkz can you review the translations [here ](https://docs.google.com/document/d/1JchTlVn4JoO710CeCh_MfikMFGqX4LV088ksjEfIHXY/edit?tab=t.0)and let me know if anything is missing?

**Comment by Charcharbinkz** [comment]
@mcman12 that looks perfect to me! FYI @ethanWallace

**Comment by SmartMouthWords** [comment]
@ethanWallace Content is ready for publishing
